packages/plugin-compiler/src/main/kotlin/io/realm/compiler/AccessorModifierIrGeneration.kt

```diff
@@ -270,8 +278,8 @@ class AccessorModifierIrGeneration(private val pluginContext: IrPluginContext) {
                         //      return backing_field
                         //  }
 
-                        // TODO: use correct getter
-                        modifyAccessor(declaration, getValue, setValue)
+                        // TODO: setter
```

Comment:
Do we allow updating the list itself?

Reply:
We talked about creating a setter that calls `clear` on the list and then `add`s whatever items the source list may have, if that's what you mean? In any case, it isn't really urgent to implement, that's why I wrote the TODO.